### PR TITLE
Add Furui Cheng's profile

### PR DESCRIPTION
**Pull request by ChengFR**

### CODE REVIEW

File: _members/furui-cheng.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+role: student
+
+services:
+  github: https://github.com/ChengFR
+
+start: Feb 2022
+end: 
+---
+[Furui Cheng](https://www.furuicheng.tech/) is a CS PhD student at the Hong Kong University of Science and Technology, advised by Prof. Huamin Qu. 
+His research interest lies in the field of **Human-Center AI** with data visualization. He designs and develops **Interactive Visualization Systems** to support users with no Machine Learning background in understanding and exploring ML models.
```
Comment (ngehlenborg) [inline _members/furui-cheng.md:17]:
```suggestion
His research interest lies in the field of **Human-Centered AI** with data visualization. He designs and develops **Interactive Visualization Systems** to support users with no Machine Learning background in understanding and exploring ML models.
```

File: _members/furui-cheng.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Furui Cheng
+name_degree: Furui Cheng, B.Eng.
+
+photo: furui-cheng.jpg
+
+job_title: Visiting Graduate Student in Computer Science
```
Comment (ngehlenborg) [inline _members/furui-cheng.md:7]:
```suggestion
job_title: Visiting Graduate Student
```

**Review by ngehlenborg — CHANGES_REQUESTED** [review]
Great! Please address the suggested edits and after that it will be ready to merge.

**Comment by ChengFR** [comment]
Thanks, Nils! I've addressed the issues.

**Pull request merged**